### DISCUSSION TITLE
Refresh Data On the Fly

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#370] [Ideas]
Don't see any option to refresh the data on the fly - ie: without restarting?  After I updated to 1.4.1 yesterday, I noticed one of my old beta db cams was glitching and stopped streaming in the wyze app. So I removed it and added another beta db cam that I used before and seemed more stable. The bridge eventually stopped trying to connect to the removed cam, but does not see the replacement. The yml contains these lines in environment, but it seemed to still want to try connecting about 8-9 times, but it appears the fresh data is only upon startup - might be good to add a repeat the fresh data check on periodic basis that can be adjusted like 60 min or daily or whatever seems logical - in general I guess adding/removing cams is not a frequent occurance so might not be worth the hassle.
- FRESH_DATA=true
- IGNORE_OFFLINE=true

**Comment by mrlt8** [comment]
I'm not sure if this is a good idea. Some of the other wyze projects on GitHub were getting rate limited, so I've actually tried to minimize the number of calls to the wyze api. As long as the camera's ENR doesn't change, you could probably go months without pinging the wyze servers.

Is there any reason why the cameras need to be added/deleted from the account? Could you just leave them all attached to the account and power them on/off as needed? That way, the list of cameras are saved locally and the bridge will just keep trying to connect to them until they are available. You could set `OFFLINE_TIME` to something like 900 so it will attempt to reconnect to any offline cameras every 15 minutes.

**Comment by cheme75** [comment]
I guess in most cases users don’t have a hand full of left over beta cams to re-use. Is there a max for the offline time? Guess that could work. You are right that normally my regular cams are always attached in my acct and online. I had 5 beta doorbells sitting around so been placing a couple of them to use to live stream some monitoring on window sills looking out at some yard areas. Just had the thought after a bit of back and forth between restarting the bridge and swapping the betas until I found the pair that seem most stable. I had been deleting the unstable units from my acct, but I guess I could leave them sitting there offline. As a side note: It’s kind of absurd that after I add one, the wyze app shoves out a cam plus trial for it but the app doesn’t respond to any of its functions other the button to ring, it just live streams - can’t even adjust sensitivity, no detection is functional. Anyway, it was just a thought.
